### DISCUSSION TITLE
Use digital PDF as proof

### DISCUSSION THREAD

**Discussion by mateusfg7** [community/community#107549] [GitHub Education] [answered]
### Hi everyone,

My university doesn't emit printed letters and papers, is all digital, and signed. Can I send a declaration dated of my registration in PDF, digitally signed by the university, as a document proof?

**Comment by faridnec** [comment]
Yes, it is acceptable. However, the better option is to take a photo of your academic ID/ proof directly with the provided camera as it is stated there.

**Reply by mateusfg7** [reply]
Thanks! At the moment, I don't have a physically academic ID yet, just digital register declaration.

**Comment by mishra-saksham** [comment, marked as answer]
@mateusfg7 Yes, you very well can share any digital media converted as an image in jpeg format and upload it but is advised to use live camera picture, so you can first try converting the pdf/email in a jpeg image and if you are accepted in the try then congratulations but if you are not wait for 5-7 days and try again this time print the proof and click a picture of it.

Hope this comment helps you and if you have any more questions I am happy to help.
I am tagging the link to the GitHub Education Official documentation for you to have a better understanding.
[GitHub Education Docs Webpage](https://docs.github.com/en/education)
[GitHub Education Apply as a student](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student)

**Reply by mateusfg7** [reply]
I was approved! Thanks.

The PDF contained two pages, I converted the PDF to JPG and merged the two pages side by side into a single image.

The image size was larger than 1MB, so I went to an image compression site to make it smaller than 1MB and submitted it. That was it, I didn't need to print and take a live picture.

**Reply by mishra-saksham** [reply]
Congratulations buddy! It is my pleasure.

**Comment by mateusfg7** [comment]
### 14 Feb, 2024 Update

I was approved! The PDF contained two pages, I converted the PDF to JPG and merged the two pages side by side into a single image.

The image size was larger than 1MB, so I went to an image compression site to make it smaller than 1MB and submitted it. That was it, I didn't need to print and take a live picture.
